dev/Repeater/APITests/RepeaterTests.cs

```diff
@@ -665,52 +666,76 @@ public void VerifyRepeaterDoesNotLeakItemContainers()
         [TestMethod]
         public void BringIntoViewOfExistingItemsDoesNotChangeScrollOffset()
         {
-            ScrollViewer scroll = null;
+            ScrollViewer scrollViewer = null;
             ItemsRepeater repeater = null;
+            AutoResetEvent scrollViewerLoadedEvent = new AutoResetEvent(false);
 
             RunOnUIThread.Execute(() =>
             {
                 repeater = new ItemsRepeater();
                 repeater.ItemsSource = Enumerable.Range(0, 100).Select(x => x.ToString()).ToList();
 
-                scroll = new ScrollViewer() {
+                scrollViewer = new ScrollViewer() {
                     Content = repeater,
                     MaxHeight = 400,
                     MaxWidth = 200
                 };
 
-                Content = scroll;
+                scrollViewer.Loaded += (object sender, RoutedEventArgs e) =>
+                {
+                    Log.Comment("ScrollViewer.Loaded event handler");
+                    scrollViewerLoadedEvent.Set();
+                };
+
+                Content = scrollViewer;
                 Content.UpdateLayout();
             });
 
             IdleSynchronizer.Wait();
 
             RunOnUIThread.Execute(() =>
             {
-                scroll.ChangeView(null, repeater.ActualHeight, null);
-                scroll.UpdateLayout();
+                Log.Comment("Scroll to end");
+                scrollViewer.ChangeView(null, repeater.ActualHeight, null);
+                scrollViewer.UpdateLayout();
             });
+
+            Log.Comment("Wait for scrolling");
+            if (Debugger.IsAttached)
+            {
+                scrollViewerLoadedEvent.WaitOne();
```

Comment:
I'm not sure I understand this change. Why are we waiting for the loaded event? We already called ChangeView on this ScrollViewer, wouldn't that have failed if it wasn't yet loaded?  Why does the loaded event signify that the scrolling is done?

Reply:
That was me blindly copy pasting old code, switched to ViewChanged now. Sorry about the confusion here.